src/analysis/raster/qgsrastercalculator.cpp

```diff
@@ -36,6 +36,41 @@
 #include "qgsgdalutils.h"
 #endif
 
+
+//
+// global callback function
+//
+int CPL_STDCALL GdalProgressCallback( double dfComplete,
+                                      const char *pszMessage,
+                                      void *pProgressArg )
+{
```

Comment:
I didn't find a lot of documentation about the `dfComplete` parameter in `GDALProgressFunc`. Can you enlighten me a little here please :)?

Reply:
Sorry, I can't :) I just copy/pasted the implementation from https://github.com/qgis/QGIS/blob/master/src/core/providers/gdal/qgsgdalprovider.cpp#L103

Reply:
Ok :)